lib/shopping_cart.rb

```diff
@@ -0,0 +1,21 @@
+class ShoppingCart
+
+  attr_reader :name, :capacity, :products
```

Comment:
The method `capacity` should return only the integer of the capacity, not the full string.